packages/assert/README.md

```diff
@@ -1,10 +1,50 @@
-assert
+@0xproject/assert
 ------
 
 Standard type and schema assertions to be used across all 0x projects and packages
 
-## Install
+## Installation
 
 ```bash
-npm install @0xproject/assert --save
+yarn add @0xproject/assert
+```
+
+## Usage
+
+TODO - Custom documentation page is in progress.
+
+## Contributing
+
+We strongly encourage our community members to help us make improvements and to determine the future direction of the protocol. To report bugs within this package, please create an issue in this repository.
+
+[CONTRIBUTING.md](../../CONTRIBUTING.md)
+
+### Install Dependencies
+
+If you don't have yarn workspaces enabled - enable them:
```

Comment:
Let's add `(enabled by default starting at Yarn v1.0)`

Reply:
Or better yet: `If you don't have yarn workspaces enabled (Yarn < v1.0) - enable them:`